gce/survival-training/dummy/train.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import glob
+import json
+import os
+import random
+
+
+def generate_trainer(hyperparameters):
+  """Generates a callable which performs a single step of training when called.
+
+  Args:
+    1. hyperparameters - hyperparameters to train with.
+
+  Returns:
+    trainer callable
+  """
+  def _trainer():
+    return random.random()
```

Comment:
why should the trainer callable return a number?  if the user needs to modify this to suit their needs, might just raise a `NotImplementedError`.  also if that is the case please specify the arguments and return type in the docstring.

Reply:
Added docstring, please have a look and see if it's acceptable.